web/components/home/Questions.js

```diff
@@ -27,11 +27,22 @@ export const QUESTIONS = Object.freeze({
       ),
     },
     {
-      'How can I officially sign up?': (
-        <p className="margin-top-none">Registration on our website will open in November.&nbsp;
-          <a href="http://eepurl.com/dF3b7n">
-            Let us know your email and we&apos;ll tell you once it does!
-          </a>
+      'Application deadline?': (
+        <p className="margin-top-none">
```

Comment:
The `<br>` looks a little sudden, but I see that the text looks too close without it. You can use `&nbsp;` between the sentences to avoid the break and still have enough space between the sentences. Making the datetimes bolded may look nice as well, I'll leave it up to you to decide.
```
        <p className="margin-top-none">
          <b>December 1, 2018 11:59PM Pacific Time</b> is the deadline for applications requesting
           travel reimbursements.&nbsp;<b>December 15, 2018 11:59PM Pacific Time</b> is the 
           deadline for the rest of applications.
        </p>
```

Reply:
I feel like it looks better with <br />

With `<br />`
![screen shot 2018-11-03 at 7 09 06 pm](https://user-images.githubusercontent.com/16379094/47959250-36ad0100-df9c-11e8-8419-fae0cd377712.png)



Using `&nbsp;`
![screen shot 2018-11-03 at 7 09 25 pm](https://user-images.githubusercontent.com/16379094/47959253-39a7f180-df9c-11e8-92c6-5cc1e22a1146.png)